forest/cli/src/cli/wallet_cmd.rs

```diff
@@ -57,7 +63,25 @@ pub enum WalletCommands {
         path: Option<String>,
     },
     /// List addresses of the wallet
-    List,
+    List {
+        /// flag to force full accuracy,
+        /// not just default 4 significant digits
```

Comment:
This is not true in the current implementation, no? There are three significant digits.
So You won't get a value like 500.2367; you will get 500.237. Unless I am missing something, let's update the documentation and add this use case (not fixed, not exact) to ensure the documentation matches the actual behaviour.
```
            (500236779800000000, "~500.237 milli FIL"),
```

Reply:
Fixed to to be 4 significant digits. Added the test case